garak/__init__.py

```diff
@@ -18,8 +17,5 @@
 
 _config.transient.log_filename = _log_filename
 
-logging.basicConfig(
-    filename=_log_filename,
-    level=logging.DEBUG,
-    format="%(asctime)s  %(levelname)s  %(message)s",
-)
+# Note: Logging configuration is performed in garak.command.start_logging()
+# to enable a process-safe Queue-based logging architecture.
```

Comment:
This not a viable revision, the setup here is intentional to capture logs from spawned processes, this will break logging when `parallel_*` options are enabled and cause logs for multiprocessing to end up on `stdout`.